core/sql/pom.xml.apache

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
```

Comment:
no idea to use one pom.xml, we have to make 3 version (vanilla,cdh,hdp), so have to use three pom.xml